packages/controls/src/controls/shape/v2/shape-v2.ts

```diff
@@ -170,11 +167,9 @@ class Definition<C extends Config> extends ControlDefinition<
   fromData(data: DataType<C> | undefined): ValueType<C> | undefined {
     if (data == null) return undefined
 
-    const versionedData = Definition.versionData(data)
+    const itemsData = Definition.itemsData(data)
```

Comment:
Here and below, use newly introduced `itemsData` method in code that works with items data.